skfda/representation/_functional_data.py

```diff
@@ -6,16 +6,15 @@
 
 from abc import ABC, abstractmethod, abstractproperty
 
-import numpy as np
-
-import matplotlib.patches as mpatches
-
-import matplotlib.pyplot as plt
 from matplotlib.axes import Axes
 import mpl_toolkits.mplot3d
 import pandas.api.extensions
 
+import matplotlib.patches as mpatches
+import matplotlib.pyplot as plt
+import numpy as np
 from skfda.representation.extrapolation import _parse_extrapolation
```

Comment:
It is not a change made in this pull request, but this import should be relative to be consistent with the rest.

```suggestion
from .extrapolation import _parse_extrapolation
```